templates/editEvents.html

```diff
@@ -137,8 +137,8 @@ <h3 class="panel-title">
                               {{/if}}
                               {{#if canAdvanceRound}}
                                 <a
-                                  href="{{pathFor 'advanceCompetitors' competitionUrlId=../../competitionUrlId eventCode=eventCode nthRound=nthRound}}"
-                                  data-toggle="tooltip" data-placement="left" data-original-title="Advance competitors to next round" data-container="body"
+                                  href="{{pathFor 'advanceParticipants' competitionUrlId=../../competitionUrlId eventCode=eventCode nthRound=nthRound}}"
+                                  data-toggle="tooltip" data-placement="left" data-original-title="Advance participants to next round" data-container="body"
```

Comment:
I imagine this data-original-title attribute has something to do with the UI? Possibly a tooltip? If so, shouldn't this still say competitor?